perfrunbook/system-load-and-compute-headroom.md

```diff
@@ -0,0 +1,157 @@
+# System Load and Compute Headroom
+
+The typical pricing model for cloud computing services centers on the amount of hardware resources set aside
+for the customer. With this, ideally such resources should be utilized fully, to operate at peak economy.
+In the ideal world the performance of a system should be linear up to 100% load.
+This can be in the context of pure compute or peak load and fail-over resilience.
+However, depending on workloads, systems that employ simultaneous multithreading (SMT), 
+aka 'Hyperthreading', incur non-linear performance degradation when the system is 
+loaded beyond 50%.
+Systems that don't use SMT will degrade as well, to some extent, because of shared resource contention.
+In the following the extent of this degradation is investigated with a few illustrative workloads.
```

Comment:
First sentence can be cut.  Can a sentence that states the conclusion also be put in here as a sentence?  For ease of reading, think of writing this to be read inside out, so the intro paragraph and conclusion paragraph can be read and give 80% of the info without needing to study the data.